### ISSUE TITLE
Change Aspect-Ratio of web-export

### ISSUE THREAD

**Issue by dxkill** [chrismaltby/gb-studio#1042]
**Is your feature request related to a problem? Please describe.**
It's not related to a problem.

**Describe the solution you'd like**
It would be cool, if there was a Feature, with which you could change the Aspect-Ratio for the web-export. for example change it to 16:9.

**Describe alternatives you've considered**
I've considered using gbsenpai, but it only works for GB Studio 2. I also tried to change the CSS/HTML files of the export. It didn't work.

**Additional context**
No additional context.

**Comment by polyinnovator** [comment]
Yeah that at the very least something like 320x180 so its still gb size sorta, but it scales in 16:9 format for modern displays

**Comment by maxoakland** [comment]
It would be stretched and look really weird. You can actually do this already by editing the emulator files

**Comment by polyinnovator** [comment]
Good to know ty

    On Sunday, January 8, 2023 at 11:27:33 AM CST, Max Oakland ***@***.***> wrote:  
 
 


It would be stretched and look really weird. You can actually do this already by editing the emulator files
